sale_coupon_domain_product_reward_fixed_price/models/sale_order.py

```diff
@@ -0,0 +1,23 @@
+from odoo import models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def _set_reward_fixed_price_for_lines(self, program):
+        """
+        Update discount field by program
+        """
+        price_unit = program.price_unit
+        lines = self._get_paid_order_lines()
```

Comment:
In this line use `_get_paid_order_lines` and than never used. For what it's here?

Reply:
Thanks, I'll fix it)